lib/routers/establishment/projects.js

```diff
@@ -97,27 +103,35 @@ router.param('id', (req, res, next, id) => {
         .select('id', 'grantedAt', 'submittedAt', 'createdAt')
         .where({ projectId: project.id })
         .orderBy('createdAt', 'desc')
-        .then(versions => {
-          const granted = versions.find(v => v.grantedAt) || null;
-          if (granted) {
-            // if a granted version exists, we're only interested in drafts created after
-            versions = versions.filter(v => v.createdAt > granted.createdAt);
-          }
-          // only attach most recent draft
-          const draft = versions[0] || null;
-          return { granted, draft };
-        })
         .then(versions => {
           req.project = {
             ...project,
-            ...versions
+            versions
           };
           next();
         });
     })
     .catch(next);
 });
 
+const canFork = (req, res, next) => {
+  const version = get(req.project, 'versions[0]');
+  if (!version) {
+    return next(new NotFoundError());
+  }
+  // version can be forked, continue
+  if (version.grantedAt || version.submittedAt) {
+    return next();
+  }
```

Comment:
Is this now incompatible with https://github.com/UKHomeOffice/asl-schema/pull/138 ?

Reply:
yes